### ISSUE TITLE
svg support

### ISSUE THREAD

**Issue by WolfgangFahl** [samwilson/diagrams-extension#13]
format='svg' doesn't seem to work anymore. I am tryint to debug this but can't even get the "types" parameter from the postData request.

**Comment by WolfgangFahl** [comment]
see http://wiki.bitplan.com/index.php/Diagrams#MediaWiki_1.33.4_workaround for a workaround. @samwilson  will this be compatible with your server? The types handling seems to be an issue you seem to try to send the value as an array and i am not sure whether that works correctly so i am only sending a single type "png" or "svg" now and the server also returns the appropriate JSON.

**Comment by samwilson** [comment]
I don't think any output format other than PNG has ever been requested by the extension from the diagrams service. See the [return value that's used in `Diagrams::renderWithService()`](https://github.com/samwilson/diagrams-extension/blob/64e3bbe97d9c55651851595b9d6e301caea90066/includes/Diagrams.php#L135).

However, it should of course be able to support this! I've added this; see #14.

**Comment by samwilson** [comment]
By the way, for the new local-rendering of graphs, SVG works correctly.

**Comment by WolfgangFahl** [comment]
Thx. So i assume:

```bash
curl -X POST -F 'generator=plantuml' -F source='vrsion' https://diagrams.bitplan.com/render
```

should return:
```json
{
  "diagrams": {
    "png": {
      "url": "https://diagrams.bitplan.com/render/png/0x97cd3329.png"
    }
  }
```
and
```bash
curl -X POST -F generator=plantuml -F source='verion' -F types='svg' https://diagrams.bitplan.com/render
```
should return
```json
{
  "diagrams": {
    "svg": {
      "url": "https://diagrams.bitplan.com/render/svg/0x97cd3329.svg"
    }
  }
}
```

and our services are comptabile. 

Hope Mediawiki 1.35 LTS will be out soon. I intend to migrate three wikifarms to that version ...

**Comment by samwilson** [comment]
Sounds good.

This issue is fixed, and released as 0.6.1.
